README.md

```diff
@@ -1,72 +1,37 @@
 # Metarhia Documentation Generator
 
-## Interface: doc
+[![TravisCI](https://travis-ci.com/metarhia/metadoc.svg?branch=master)](https://travis-ci.com/metarhia/metadoc)
 
-### Introspect interface
+## Installation
 
-`introspect(namespace, text)`
-  - `namespace: `[`<Map>`] hash of interfaces
-  - `text: `[`<string>`] data to parse
+```bash
+npm install https://github.com/metarhia/metadoc
```

Comment:
Shouldn't this be `npm install @metarhia/doc`?
I assume we will publish the next version (0.1.0) as @metarhia/doc.

Reply:
I think this should be changed when it is released. Referring to the unexisting package can be misleading.